packages/button/src/Button.spec.tsx

```diff
@@ -143,6 +135,17 @@ describe('packages/button', () => {
       expect(onClick.mock.calls.length).toBe(0);
     });
 
+    test('does not fire onClick handler on disabled anchor', () => {
+      const onClick = jest.fn();
+      const { button } = renderButton({
+        disabled: true,
+        as: 'a',
+        onClick,
+      });
+      fireEvent.click(button);
+      expect(onClick.mock.calls.length).toBe(0);
```

Comment:
I realize this is following the pattern of the previous test, but I think `expect(onClick).toHaveBeenCalledTimes(0)` is a bit cleaner